src/main/java/bio/terra/service/auth/iam/IamAction.java

```diff
@@ -36,13 +36,15 @@ public enum IamAction {
   // snapshots
   UPDATE_SNAPSHOT,
   READ_DATA,
+  // journal
```

Comment:
Ah good catch

Reply:
I'm not really sure why I reordered these! I'll move it back.